services/ui-src/src/types/index.ts

```diff
@@ -41,6 +41,12 @@ export interface ReportPath {
   children?: ReportPath[];
 }
 
+export enum ReportStatus {
+  INITIAL = "",
+  IN_PROGRESS = "In Progress",
+  COMPLETED = "Completed",
```

Comment:
We should double-check with David what these statuses should be. For now, let's go with the following and make any necessary changes to the other files to align.
```suggestion
  CREATED = "Created",
  IN_PROGRESS = "In Progress",
  SUBMITTED = "Submitted",
```